docs/slurm-dws-flex.md

```diff
@@ -13,25 +13,19 @@ With Dynamic Workload Scheduler in Flex Start mode, you submit a GPU capacity re
 > The project needs to be allowlisted for private preview access.
 > Fill out the [form](https://docs.google.com/forms/d/1etaaXMW9jJUTTxfUC7TIIMttLWT5H-3Q8_3-sG6vwKk/edit).
 
-In order to make use of DWS Flex Start mode with SlurmGCP, you must specify a proper set of `instance_properties` in the `schedmd-slurm-gcp-v6-nodeset` module. See the example below:
+In order to make use of DWS Flex Start mode with SlurmGCP, you must use the `dws_flex` variable in the `schedmd-slurm-gcp-v6-nodeset` module. From there you can specify the desired maximum duration (in seconds) with `max_run_duration`. See the example below:
 
 ```yaml
   - id: flex_nodeset
     source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
     use: [network]
     settings:
-      instance_properties:
-        reservationAffinity:
-          consumeReservationType: NO_RESERVATION
-        scheduling:
-          maxRunDuration: { seconds: $(2 * 60 * 60) } # 2 hours
-          onHostMaintenance: TERMINATE
-          instanceTerminationAction: DELETE
+       dws_flex:
+          enabled: true
```

Comment:
nit. We can omit `enabled` to appear more slick ))

Reply:
Sure, I also added `enable_placement: false` since that will be needed for nodesets using dws_flex